base/genericmemory.jl

```diff
@@ -145,10 +151,9 @@ end
 
 ## Conversions ##
 
-convert(::Type{T}, a::AbstractArray) where {T<:GenericMemory} = a isa T ? a : T(a)::T
+convert(::Type{T}, a::AbstractArray) where {T<:Memory} = a isa T ? a : T(a)::T
```

Comment:
should the second argument here be `AbstracVector`?

Reply:
apparently not?
```
julia> @which convert(Vector{Int}, [1 2; 3 4])
convert(::Type{T}, a::AbstractArray) where T<:Array
     @ Base array.jl:607

julia> convert(Vector{Int}, [1 2; 3 4])
ERROR: MethodError: no method matching Vector{Int64}(::Matrix{Int64})
```